crates/iota-indexer/migrations/pg/2025-03-13-081552_optimistic_transactions/up.sql

```diff
@@ -0,0 +1,90 @@
+CREATE TABLE tx_insertion_order (
```

Comment:
Please add a description to each table definition to understand what the table is doing and why it is needed.

Reply:
Done